File: module_user/src/main/java/com/yyxnb/module_user/ui/UserActivity.java
package com.yyxnb.module_user.ui;

import androidx.fragment.app.Fragment;

import com.alibaba.android.arouter.facade.annotation.Route;
import com.yyxnb.common_base.core.ContainerActivity;

import static com.yyxnb.common_res.arouter.ARouterConstant.USER_MAIN;

@Route(path = USER_MAIN)
public class UserActivity extends ContainerActivity {

    @Override
    public Fragment initBaseFragment() {
        return new UserFragment();
    }
}

File: module_user/src/main/java/com/yyxnb/module_user/ui/UserActivity.kt
package com.yyxnb.module_user.ui

import androidx.fragment.app.Fragment
import com.alibaba.android.arouter.facade.annotation.Route
import com.yyxnb.common_base.arouter.ARouterConstant.USER_ACTIVITY
import com.yyxnb.common_base.base.ContainerActivity

@Route(path = USER_ACTIVITY)
class UserActivity : ContainerActivity() {

    override fun initBaseFragment(): Fragment? {
        return UserFragment()
    }
}